stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/composer/MessageComposer.kt

```diff
@@ -77,6 +78,7 @@ public fun MessageComposer(
     val value by viewModel.input.collectAsState()
     val selectedAttachments by viewModel.selectedAttachments.collectAsState()
     val activeAction by viewModel.lastActiveAction.collectAsState(null)
+    val maxMessageLength by viewModel.maxMessageLength.collectAsState()
```

Comment:
Should we expose `MessageInputState` in the ViewModel?

Reply:
I was thinking about it, but there's no pretty way to do it - we'd have to combine multiple flows and use `combine/combineLatest` on them to form a composite flow that creates and updates the `MessageInputState`.

If you feel like this is something that's worthwhile, we can do it in this PR and then update our samples & docs.

WDYT?